redaxo/src/core/lib/login/login.php

```diff
@@ -309,6 +311,30 @@ public static function startSession()
     }
 
     /**
+     * Einstellen der Cookie Paramter bevor die session gestartet wird
+     */
+    private static function setCookieParams() {
+        $cookieParams = session_get_cookie_params();
+
+        $key = rex::isBackend() ? 'backend' : 'frontend';
+        $sessionConfig = rex::getProperty('session');
+
+        foreach($sessionConfig[$key]['cookie'] as $name => $value) {
+            if ($value != null) {
```

Comment:
Würde `!==` nehmen. Versuche in neuem Code eigentlich wo immer es geht, mit den typsicheren Vergleichsops zu arbeiten.

Reply:
done